crates/astria-bridge-contracts/src/lib.rs

```diff
@@ -420,6 +420,9 @@ where
             timeout_time: timeout_in_5_min(),
             source_channel,
             bridge_address: Some(self.bridge_address),
+            // FIXME: this needs a way to determine when to use compat address
+            // https://github.com/astriaorg/astria/issues/1424
+            use_compat_address: false,
```

Comment:
we could potentially check if the denom contains the `usdc` string in it

Reply:
@noot Yeah, I am thinking about how to do this. Can you add that to the issue I linked? We should probably come up with a PRD.